### DISCUSSION TITLE
links

### DISCUSSION THREAD

**Discussion by giscus** [tangly1024/tangly1024#53] [Announcements]
# links

站友们的集合

https://tangly1024.com/links

**Comment by tangly1024** [comment]
因评论区访问量过大，暂时关闭其它渠道，只保留giscus。

欢迎大家在评论区交换友链！

**Reply by 712346867** [reply]
期待加入你的友情链接列表

名称：小太阳昌仔BLOG
签名：平静的努力，精致的执著
头像：[头像地址](https://www.notion.so/image/https%3A%2F%2Fs3-us-west-2.amazonaws.com%2Fsecure.notion-static.com%2Fb28fbe79-4931-4645-84c3-35297e9ff340%2Fnotion%25E5%25A4%25AA%25E9%2598%25B3%25E5%259B%25BE%25E6%25A0%2587.gif?table=collection&id=88b40fd2-0e9f-445b-893d-c2d391062edd&width=200)

**Comment by DuckDuck88** [comment]
期待互链接，[地址](yaya.run)
名称：YaYa‘s Home
头像：[头像地址](https://markdown-mac-work-1306720256.cos.ap-guangzhou.myqcloud.com/jpg/new.jpg)

**Reply by tangly1024** [reply]
欢迎~

**Comment by cativen** [comment]
期待互链接，https://www.codezhou.top/

**Reply by tangly1024** [reply]
欢迎~

**Comment by DuckDuck88** [comment]
> 期待互链接，https://www.codezhou.top/

大佬，你这里的点击跳转其他网址是怎么做的呀？
<img width="323" alt="image" src="https://github.com/tangly1024/tangly1024/assets/45893278/ec7daffd-aaf6-494f-899d-aa945ec48715">

**Reply by LH1228** [reply]
在slug栏目填写链接

**Reply by DuckDuck88** [reply]
感谢~

**Comment by 1985312383** [comment]
来了来了，我终于搭好并且写好东西了
https://www.kemuling.blog/

**Reply by tangly1024** [reply]
欢迎！~

**Comment by LH1228** [comment]
兄弟们，Twikoo评论插件是不是要给个域名才能直连评论呀？

**Reply by tangly1024** [reply]
最好整个域名，不然国内可能访问不了

**Comment by dishengai** [comment]
签到 感谢提供这么好的项目哈哈 之前一直不知道怎么部署(别的项目好难好复杂)这个相对非常简单 新手非常友好哦
迪升的博客
blog.disheng.org

**Reply by dishengai** [reply]
https://blog.disheng.org/

**Reply by dishengai** [reply]
迪升的博客https://blog.disheng.org/

**Comment by macroCyborg** [comment]
期待互链😄，作者的项目搭建notion内容管理太方便了，简直丝滑。 
头像=>![](https://c1ns.cn/HQmAL)
签名: 一只猫
链接:https://macrocyborg.site

**Reply by macroCyborg** [reply]
头像=> https://c1ns.cn/HQmAL

**Comment by maple-dawn** [comment]
期待互链接：https://www.imaple.tech/
名称：晨枫Maple Tech
头像：https://www.notion.so/image/https%3A%2F%2Fs3-us-west-2.amazonaws.com%2Fsecure.notion-static.com%2F7f394fdd-5b43-4a1f-8e5d-cbb6d71fc372%2FMaple_Hayao_Miyazaki_animation_style_d3672b1d-6c4a-4de0-89e0-9a0eaad7c425.png?table=collection&id=5260cbc9-52c2-48b7-a20c-c54523700988&width=400

**Comment by sloth2012** [comment]
谢谢大佬的项目，非常棒：
期待互链：https://dage110.com/
名称：哒鸽

**Comment by liano3** [comment]
大佬太强了！本学生党原来的博客部署在github上访问速度很慢😢。最近才偶然发现这个宝藏项目，在不提升成本的情况下速度嘎嘎快，更新嘎嘎方便，泰酷辣😘
网址：https://blog.liano.top
名称：Liano-Blog
描述：要求高，水平低，擅长睡觉，懒得社交
图片：https://blog.liano.top/bg_image.jpg
期待互链~🙏🏻

**Comment by f1shQAQ** [comment]
大佬太强了，希望能互换友链
名称: 咸鱼小窝
描述: 万事胜意 平安喜乐。
链接: https://qqquq.com/

**Reply by f1shQAQ** [reply]
图标: https://bu.dusays.com/2023/12/04/656d6235635ef.webp

**Comment by vsd269** [comment]
期待互链接：https://www.ai-hd.com/
名称：Ai-皇帝
描述：分享AIGC与实用技巧
图片：https://tupian1014-1251479570.cos.ap-shanghai.myqcloud.com/tupian/Ai-%E7%9A%87%E5%B8%9D.jpg

**Comment by EachenKuang** [comment]
期待互换友链！！
站点名称：易浅小站
站点描述：青春不老，我们不散。永远年轻，永远热泪盈眶。
站点地址：https://kuangyichen.com
站点图标：https://image.kuangyichen.com/image/eachen_icon.png

**Comment by RayDream3** [comment]
期待互换友链！！
站点名称：云烟阁
站点描述：瑞烟起，彩云行
站点地址：https://raydream.top
站点图标：https://www.notion.so/image/https%3A%2F%2Fprod-files-secure.s3.us-west-2.amazonaws.com%2F9f9bb174-699f-4e35-8289-6d7d2578ce7e%2Ffd3427c0-4662-4ba4-a986-b62704668740%2F847118623.jpeg?id=db25c8ca-ea77-48ff-851f-defa2f396438&table=collection&spaceId=9f9bb174-699f-4e35-8289-6d7d2578ce7e&width=60&userId=adb476bc-c026-40f6-ae6a-653277a9b209&cache=v2

**Comment by EachenKuang** [comment]
您好，您的邮件已经收到。我会很快回复的！Hello, your email has been received. I will reply soon!

**Comment by robinhuang123456** [comment]
期待互链接：https://www.ainio.fun/
名称：ainio
描述：chatgpt AI 新闻 ，教程分享，数据交易

**Comment by Wcowin** [comment]
大佬，我们可以交换一下友链~
>名称: Wcowin's Web  
>链接: https://wcowin.work/  
>头像: https://s2.loli.net/2024/02/01/gaE47y5fKM6kosV.png  
>简介: 循此苦旅，以达星辰
